packages/gen-typescript-declarations/src/test/goldens/paper-behaviors-3/paper-button-behavior.d.ts

```diff
@@ -8,6 +8,8 @@
  *   paper-button-behavior.js
  */
 
+export {PaperButtonBehaviorImpl};
```

Comment:
Does this resolve to anything?

Reply:
Yep, only identifiers that can be resolved are included. I'm not sure why we currently export the "Impl" behavior objects, but that's what modulizer did.

Reply:
Oh I suppose you mean does it resolve to a valid TypeScript symbol? No, it doesn't. I'll need to address that later. A lot of funky things going on here too, e.g. the way namespaces are handled.

Reply:
It turns out that we use both behaviors and their impl objects in different places in the elements, so that's correct and load bearing.